.github/workflows/build.yml

```diff
@@ -19,6 +19,9 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
+        include:
+          - os: ubuntu-latest
+            full-build: true
```

Comment:
Similar to the example here:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations